torch/csrc/dynamo/python_compiled_autograd.cpp

```diff
@@ -85,6 +85,9 @@ static void check(bool result) {
     check(nullptr);
 }
 
+// snapshot of automatic custom opification of CppNodes
+static bool opaque_cpp_node;
```

Comment:
Maybe we could find a cleaner way to pass configs to compile autograd?

C++ globals seem not ideal:
1) They are duplicates of stuff in Python (this PR doesn't properly keep them in sync, but even if it did keeping them in sync would be error prone)
2) They aren't thread local, so if we try to support multithreading in the future this will be harder
3) I think we need to hash on them so changing these values recompiles?